editor/editor_node.cpp

```diff
@@ -1265,7 +1265,12 @@ void EditorNode::_dialog_action(String p_file) {
 			ProjectSettings::get_singleton()->set("application/run/main_scene", p_file);
 			ProjectSettings::get_singleton()->save();
 			//would be nice to show the project manager opened with the highlighted field..
-			_run(false, ""); // automatically run the project
+
+			if ((bool)pick_main_scene->get_meta("from_native")) {
```

Comment:
Shouldn't you call `has_meta("from_native")` first?

Reply:
Fixed.